src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -160,18 +162,27 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
           if (accessLevel < WorkspaceAccessLevels.Read)
             Future.successful(RequestComplete(ErrorReport(StatusCodes.NotFound, noSuchWorkspaceMessage(workspaceName))))
           else {
-            gcsDAO.getOwners(workspaceContext.workspace.workspaceId) map { owners =>
-              val response = WorkspaceListResponse(accessLevel,
-                workspaceContext.workspace,
-                getWorkspaceSubmissionStats(workspaceContext, txn),
-                owners)
-              RequestComplete(StatusCodes.OK, response)
-            }
+            val response = WorkspaceListResponse(accessLevel,
+              workspaceContext.workspace,
+              getWorkspaceSubmissionStats(workspaceContext, txn),
+              getWorkspaceOwners(workspaceName))
+            Future.successful(RequestComplete(StatusCodes.OK, response))
           }
         }
       }
     }
 
+  def getWorkspaceOwners(workspaceName: WorkspaceName): Seq[String] =
+    dataSource.inTransaction(readLocks=Set(workspaceName)) { txn =>
+      val ownerGroup = containerDAO.authDAO.loadGroup(RawlsGroupRef(
+        RawlsGroupName(UserAuth.toWorkspaceAccessGroupName(workspaceName, WorkspaceAccessLevels.Owner))), txn).getOrElse(
```

Comment:
Here's another simplification function: `'RawlsGroup(workspaceName, WorkspaceAccessLevels.Owner)` is all you need.